features/admin.authentication-flow-builder.v1/components/side-panel-drawer.scss

```diff
@@ -17,7 +17,7 @@
  */
 
 :root {
-    --asg-auth-flow-builder-side-panel-drawer-width: 450px;
+    --asg-auth-flow-builder-side-panel-drawer-width: 480px;
```

Comment:
Shall we add a screenshot of the UI with this change?

Reply:
<img width="1330" alt="Screenshot 2025-01-28 at 21 02 17" src="https://github.com/user-attachments/assets/064719e3-36f1-41ab-a0bb-ec0a9a28e253" />